source/contracts/Controller.sol

```diff
@@ -127,26 +124,11 @@ contract Controller is IController {
         return true;
     }
 
-    function switchModeSoOnlyEmergencyStopsAndEscapeHatchesCanBeUsed() public devModeOwnerOnly returns (bool) {
+    function switchToCentralizedMode() public devModeOwnerOnly returns (bool) {
```

Comment:
Should this be `switchToDecentralizedMode`?  Or, a better name might be something like, `switchOffDevMode`.